packages/ethereum/sdk/src/common/test/index.ts

```diff
@@ -0,0 +1,15 @@
+import {
+	getPromiEventConfirmationPromise,
+	getPromiEventHashPromise,
+} from "@rarible/web3-v4-ethereum/build/utils/to-promises"
+
+
+export async function sentTx(source: any, options: any): Promise<string> {
```

Comment:
we need to add typings here, is it possible? Even if it's internal code we're writing tests in typescript